HarpGuru/HarpFace/HarpRow/types/index.ts

```diff
@@ -1,11 +1,14 @@
 import type { ViewStyle } from 'react-native'
 
-import type { HarpFaceProps } from '../../types'
+import type { HarpCellProps } from '../../HarpCell'
 
+// TODO: this should probably be defined in HarpCell
+// since that's the lowest level at which we use Coords.
```

Comment:
Link to this comment in https://github.com/js-jslog/harpnative/issues/31 and then delete this TODO

Reply:
Done